cloud_governance/main/main.py

```diff
@@ -116,13 +116,18 @@ def run_policy(account: str, policy: str, region: str, dry_run: str):
         end_date = os.environ.get('end_date', '')
         granularity = os.environ.get('granularity', '')
         file_name = os.environ.get('file_name', '')
+        account = os.environ.get('account', '')
+        if account:
+            if account == 'perf':
```

Comment:
Why we check only for perf account, what about other accounts ?

Reply:
We are using a short-form perf-dept as perf, I didn't disturb the flow.